src/commonMain/kotlin/org/kosat/CDCL.kt

```diff
@@ -62,141 +61,49 @@ class CDCL(val clauses: MutableList<MutableList<Int>>, initNumber: Int = 0) {
         }.sortedBy { litIndex(it) }.filter { litIndex(it) > 0 }
 
     // values of variables
-    private val vars: MutableList<VarState> = MutableList(varsNumber + 1) { VarState(VarStatus.UNDEFINED, -1, -1) }
+    protected val vars: MutableList<VarState> = MutableList(varsNumber + 1) { VarState(VarStatus.UNDEFINED, -1, -1) }
 
     // all decisions and consequences
-    private val trail: MutableList<Int> = mutableListOf()
+    protected val trail: MutableList<Int> = mutableListOf()
 
     // decision level
```

Comment:
Current decision level
("current" because it is `var`, and it naturally changing overtime)